frontend/src/components/NotificationsBar/Notification/Notification.tsx

```diff
@@ -12,16 +12,24 @@ interface NotificationProps {
 
 const NotificationPlate: React.FC<NotificationProps> = ({ note }) => {
   const isOldNotification = note.status === NOTIFICATION_STATUSES.old;
+  const isUserNotifications = note.type === NOTIFICATION_TYPES.user;
 
   return (
     <NotificationContainer isOld={isOldNotification}>
       <ImageWrapper>
         <img alt="notification" src={isOldNotification ? warningIcon : redWarningIcon} />
       </ImageWrapper>
-      <div>
-        <Title>{note.title}</Title>
-        <Description>{note.description}</Description>
-      </div>
+      {isUserNotifications ? (
```

Comment:
for what you are changed layout? You can change only content inside Title and Description